### ISSUE TITLE
nics unavailable when testnic specified in configure

### ISSUE THREAD

**Issue by rainey** [appneta/tcpreplay#474]
This is maybe the same issue initially reported in bug #178 - Based on the responses to that, it seems the intention is for the --with-testnic configure option to not affect the main programs, but it does.

Having a look through the configure script, it seems that when the --with-testnic option is specified, ENABLE_PCAP_FINDALLDEVS will become undefined, and the resulting builds will not have the --listnics option available, and for example in src/tcpreplay_api.c you have this, which causes no interfaces to be available to tcpreplay.

```
#ifdef ENABLE_PCAP_FINDALLDEVS  
   ctx->intlist = get_interface_list();  
#else  
  ctx->intlist = NULL;  
#endif
```

I'm testing this with arch linux, with pcap version 1.8.1 installed.

**Comment by rainey** [comment]
Fixing this in configure.ac was fairly straightforward.  Should I submit a pull request?

**Comment by fklassen** [comment]
👍

**Comment by fklassen** [comment]
Merged to branch appneta:Bug_#474_enable_pcap_findalldevs_with_testnic but not merging to 4.3. Instead moving forward with fix for #178.
